articles/aks/operator-best-practices-cluster-security.md

```diff
@@ -53,6 +53,10 @@ For more information about Azure AD integration, Kubernetes RBAC, and Azure RBAC
 >
 > Add a network policy in all user namespaces to block pod egress to the metadata endpoint.
 
+> [!Note]
```

Comment:
Needs to be reformatted to - [!NOTE] (All uppercase).

Reply:
Completed